CMakeLists.txt

```diff
@@ -164,6 +164,10 @@ if(OPENSSL_FOUND)
     target_link_libraries(${PROJECT_NAME} ${OPENSSL_LIBRARIES})
 endif()
 
+if(WIN32)
```

Comment:
Thanks for pointing out the MinGW environment. I would be glad to add it to the list of the supported environments. I see that CMake recognizes the `MINGW` variable and thus `WIN32` could be replaced. Do you mind to change that part?